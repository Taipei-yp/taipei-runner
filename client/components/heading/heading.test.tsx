import React from "react";
import TestRenderer from "react-test-renderer";
import { Heading } from ".";

describe("Component Heading", () => {
  const heading = TestRenderer.create(
    <Heading text="testtext" color="primary" size="l" className="testclass" />,
  );

  it("correct render", () => {
    expect(heading.toJSON()).toMatchSnapshot();
  });
  it("correct props", () => {
    expect(heading.root.props.color).toBe("primary");
    expect(heading.root.props.size).toBe("l");
    expect(heading.root.props.className).toBe("testclass");
  });
});
