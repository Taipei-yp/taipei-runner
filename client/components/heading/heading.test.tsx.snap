// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Heading correct render 1`] = `
<h1
  className={
    Object {
      "has": [Function],
      "is": [Function],
      "mix": [Function],
      "split": [Function],
      "state": [Function],
      "toString": [Function],
    }
  }
>
  testtext
</h1>
`;
